src/Microsoft.ML.Data/DataLoadSave/CompositeDataLoader.cs

```diff
@@ -76,47 +113,27 @@ public void SaveTo(IHostEnvironment env, Stream outputStream)
                 using (var rep = RepositoryWriter.CreateNew(outputStream, ch))
                 {
                     ch.Trace("Saving data loader");
-                    ModelSaveContext.SaveModel(rep, Loader, "Reader");
+                    ModelSaveContext.SaveModel(rep, Loader, LoaderDirectory);
 
                     ch.Trace("Saving transformer chain");
-                    ModelSaveContext.SaveModel(rep, Transformer, TransformerChain.LoaderSignature);
+                    ModelSaveContext.SaveModel(rep, Transformer, TransformerDirectory);
                     rep.Commit();
                 }
             }
         }
-    }
 
-    /// <summary>
-    /// Utility class to facilitate loading from a stream.
-    /// </summary>
-    [BestFriend]
-    internal static class CompositeDataLoader
-    {
-        /// <summary>
-        /// Save the contents to a stream, as a "model file".
-        /// </summary>
-        public static void SaveTo<TSource>(this IDataLoader<TSource> loader, IHostEnvironment env, Stream outputStream)
-            => new CompositeDataLoader<TSource, ITransformer>(loader).SaveTo(env, outputStream);
+        internal const string Summary = "A loader that encapsulates a loader and a transformer chain.";
```

Comment:
>A loader that encapsulates a loader [](start = 41, length = 35)

Maybe: 'A model loader that encapsulates a text loader and a transformer chain". so it sounds less tautological.